modules/training_pipeline/training_pipeline/api/training.py

```diff
@@ -112,28 +108,31 @@ def train(self) -> SFTTrainer:
             train_dataset=self._training_dataset,
             eval_dataset=self._validation_dataset,
             peft_config=self._peft_config,
-            dataset_text_field="text",
+            dataset_text_field="prompt",
             max_seq_length=self._max_seq_length,
             tokenizer=self._tokenizer,
             args=self._training_arguments,
             packing=True,
             compute_metrics=self.compute_metrics,
         )
-        trainer.train()
-
-        best_model_checkpoint = trainer.state.best_model_checkpoint
-        has_best_model_checkpoint = best_model_checkpoint is not None
-        if has_best_model_checkpoint:
-            best_model_checkpoint = Path(best_model_checkpoint)
-            logger.info(
-                f"Logging best model from {best_model_checkpoint} to the model registry..."
-            )
-
-            self.log_model(best_model_checkpoint)
-        else:
-            logger.warning(
-                "No best model checkpoint found. Skipping logging it to the model registry..."
-            )
+        try:
+            trainer.train()
+
+            best_model_checkpoint = trainer.state.best_model_checkpoint
+            has_best_model_checkpoint = best_model_checkpoint is not None
+            if has_best_model_checkpoint:
+                best_model_checkpoint = Path(best_model_checkpoint)
+                logger.info(
+                    f"Logging best model from {best_model_checkpoint} to the model registry..."
+                )
+
+                self.log_model(best_model_checkpoint)
+            else:
+                logger.warning(
+                    "No best model checkpoint found. Skipping logging it to the model registry..."
+                )
+        except Exception as e:
```

Comment:
Sorry, but I don't like catching any type of exception (unless you have some specific exceptions you want to catch). I think letting the program crash and seeing all the information available about the error is better.

Reply:
Fair enough, will fix. 
We could add error handling towards the end-to-end impl.

Reply:
Sounds good 👍🏻